### PR TITLE
Import sar data files

### PR DESCRIPTION
**Pull request by m0dular**

### CODE REVIEW

File: files/Postgresql_performance.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 1,
-  "id": 38,
+  "id": 3,
```
Comment (bastelfreak) [inline files/Postgresql_performance.json:27]: are the changee here actually related to the PR?

Comment (m0dular) [reply files/Postgresql_performance.json:27]: I should have made a note about that, but the current Postgres dashboard has a misconfig where there's a default selection for the database sizes panel, so nothing shows up by default.  I snuck that fix in here.

Comment (bastelfreak) [reply files/Postgresql_performance.json:27]: I think it makes sense to pull that into another PR. you are now mixing an rnhancement with a bugfix in the same PT and commit. so reverting the enhancement will introduce a regression. And the dashboard fiy wont show up in the changelog.

Comment (m0dular) [reply files/Postgresql_performance.json:27]: Agreed, I took it out of this PR.

**Review by MartyEwings — APPROVED** [review]

**Pull request merged**